ee/tabby-webserver/src/service/background_job/mod.rs

```diff
@@ -101,14 +138,16 @@ pub async fn start(
                         continue;
                     }
 
-                    let logger = JobLogger::new(db.clone(), job.id);
-                    debug!("Background job {} started, command: {}", job.id, job.command);
+                    let job_id = job.id;
+                    let logger = JobLogger::new(db.clone(), job_id);
+                    debug!("Background job {} started, command: {}", job_id, job.command);
```

Comment:
i didn’t see any meaningful change here, consider revert?

Reply:
job_id is used later in L195 `notify_job_error!(notification_service, err, event_clone, job_id);`,
the use of `job.id` will move the value out and can no be used in L195

Reply:
i don’t see a reason using macro anymore. Any reason we cannot make it a function?

Reply:
change to a function would also handle this lifetime change, let me have a try